src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -448,7 +448,7 @@ extern "C" {
 #define SERVICE_CALL_CONTEXT_CURRENT_VERSION                0
 #define STREAM_DESCRIPTION_CURRENT_VERSION                  1
 #define FRAGMENT_ACK_CURRENT_VERSION                        0
-#define STREAM_METRICS_CURRENT_VERSION                      0
+#define STREAM_METRICS_CURRENT_VERSION                      1
 #define CLIENT_METRICS_CURRENT_VERSION                      0

```

Comment:
No need to update client metrics version?

Reply:
no - haven't changed that structure